README.md

```diff
@@ -6,22 +6,91 @@
+monolithic architecture, Zebra has a modular, library-first design, with the
+intent that each component can be independently reused outside of the `zebrad`
+fullnode.  For instance, the `zebra-network` crate containing the network stack
+can also be used to implement anonymous transaction relay, network crawlers, or
+other functionality, without requiring a full node.
 
-Most of our work so far has gone into `zebra-network`, building a new
-networking stack for Zcash, `zebra-chain`, building foundational data
-structures, `zebra-consensus`, implementing consensus rules, and
-`zebra-state`, providing chain state.
+Our first goal is to be able to participate in the network and replicate the
+Zcash chain state, and we intend to ship an alpha before the end of 2020 with
+this functionality.  In 2021, we intend to add RPC support and wallet
+integration.  This phased approach allows us to test the independent
+implementation of the consensus rules before asking users to entrust it with
+their funds.
 
-[Zebra Website](https://zebra.zfnd.org).
+At a high level, the fullnode functionality required by `zebrad` is factored
+into several components:
 
-[Rendered docs from the `main` branch](https://doc.zebra.zfnd.org).
+- [`zebra-chain`](https://doc.zebra.zfnd.org/zebra_chain/index.html), providing
+  definitions of core data structures for Zcash, such as blocks, transactions,
+  addresses, etc., and related functionality.  It also contains the
+  implementation of the consensus-critical serialization formats used in Zcash.
+  The data structures in `zebra-chain` are defined to enforce [*structural
+  validity*](https://zebra.zfnd.org/dev/rfcs/0002-parallel-verification.html#verification-stages) by making invalid states unrepresentable.  For instance, the
+  `Transaction` enum has variants for each transaction version, and it's
+  impossible to construct a transaction with, e.g., spend or output
+  descriptions but no binding signature, or, e.g., a version 2 (Sprout)
+  transaction with Sapling proofs.  Currently, `zebra-chain` is oriented
+  towards verifying transactions, but will be extended to support creating them
+  in the future.
 
-[Join us on Discord](https://discord.gg/na6QZNd).
+- [`zebra-network`](https://doc.zebra.zfnd.org/zebra_network/index.html),
+  providing an asynchronous, multithreaded implementation of the Zcash network
+  protocol inherited from Bitcoin.  In contrast to `zcashd`, each peer
+  connection has a separate state machine, and the crate translates the
+  external network protocol into a stateless, request/response-oriented
+  protocol for internal use.  The crate provides two interfaces: an
+  auto-managed connection pool that load-balances requests over available
+  peers, and a `connect_isolated` method that produces a peer connection
+  completely isolated from all other node state.  This can be used, for
+  instance, to safely relay data over Tor, without revealing distinguishing
+  information.
```

Comment:
Should we also mention inbound requests here?

Reply:
Not sure, I thought that it fit better in the `zebrad` section, since that's where inbound requests are actually implemented, and I didn't want to mention anything twice to keep it as concise as possible without losing information.